WordPress/gutenberg
# [Inserter]: Remove `Reusable` tab and move the contents inside `Blocks` tab

## What?
Remove Reusable tab and move the contents inside Blocks tab



## Testing Instructions
1. Have at least one `reusable block`.
2. Open the inserter and observe that `Reusable` tab is not there
3. In `Blocks` tab the reusable blocks should be shown as the end of the core categories(after `embed` category)

## Screenshots or screencast 

https://user-images.githubusercontent.com/16275880/202446585-33c761dc-acf4-4a58-80a5-a3de78249e7a.mov


## Notes 
If we go with this approach, we will need feedback/testing from the mobile team.
